mscore/dragelement.cpp

```diff
@@ -98,7 +98,8 @@ void ScoreView::doDragElement(QMouseEvent* ev)
 
       for (Element* e : sel.elements()) {
             QVector<QLineF> elAnchorLines = e->dragAnchorLines();
-            const QPointF pageOffset(e->findAncestor(ElementType::PAGE)->pos());
+            Element* el = e->findAncestor(ElementType::PAGE);
```

Comment:
I suggest renaming `el` to `page` and making it `const`:

`Element* const page = e->findAncestor(ElementType::PAGE);`

Reply:
much better name indeed